Analysis/Tasks/SkimmingTutorials/spectraNucleiProvider.cxx

```diff
@@ -49,7 +49,10 @@ struct NucleiSpectraProviderTask {
   Filter collisionFilter = nabs(aod::collision::posZ) < vertexZCut;
   Filter trackFilter = (nabs(aod::track::eta) < trackEtaCut) && (aod::track::isGlobalTrack == (uint8_t) true) && (aod::track::pt > trackPtCut);
 
-  using TrackCandidates = soa::Filtered<soa::Join<aod::Tracks, aod::TracksExtra, aod::pidRespTPC, aod::pidRespTOF, aod::pidRespTOFbeta, aod::TrackSelection>>;
+  using TrackCandidates = soa::Filtered<soa::Join<aod::Tracks, aod::TracksExtra,
```

Comment:
@nzardosh I now move from the full table to the split one by default, this is transparent for you, on the pidTOF and pidTPC side, e.g. `o2-analysis-pid-tof --pid-pi 1 --pid-pr 1` This is to let you know, maybe there are even more particle species that can be removed in the future.
The same applies to the other tasks.